crates/eip7702/src/auth_list.rs

```diff
@@ -92,8 +92,13 @@ impl Authorization {
     }
 
     /// Convert to a signed authorization by adding a signature.
-    pub const fn into_signed(self, signature: Signature) -> SignedAuthorization {
-        SignedAuthorization { inner: self, signature }
+    pub fn into_signed(self, signature: Signature) -> SignedAuthorization {
+        SignedAuthorization {
+            inner: self,
+            r: signature.r(),
+            s: signature.s(),
+            y_parity: U8::from(signature.v().y_parity() as u8),
```

Comment:
I believe this will force convert any v value to 0,1, which I think is fine?

Reply:
yep, we only care about boolean part of v here